app/controllers/admin/teachers/reopen_induction_controller.rb

```diff
@@ -2,7 +2,11 @@ module Admin
   module Teachers
     class ReopenInductionController < AdminController
       def update
-        @teacher = Teacher.find(params[:teacher_id])
+        @teacher = Teacher
+          .includes(:last_induction_period)
```

Comment:
We're always using the `last_induction_period`, so I thought it made sense to eager load it 👌🏼